package-lock.json

```diff
@@ -29,7 +29,7 @@
         "strip-json-comments": "3.1.1",
         "supports-color": "8.1.1",
         "which": "2.0.2",
-        "workerpool": "6.1.5",
+        "workerpool": "^6.2.0",
```

Comment:
This version should be pinned to 6.2.0. Could rebase to `master` and rebuild `package-lock.json`, please?
Then run a force push to this PR.

Reply:
done!